src/main/scala/org/allenai/plugins/DockerBuildPlugin.scala

```diff
@@ -426,6 +510,26 @@ $DOCKERFILE_SIGIL
     imageDirectory
   }
 
+  /** Task to build the main docker image for the project. This returns the image ID. */
+  lazy val mainImageBuildDef: Def.Initialize[Task[String]] = Def.task {
+    // This task requires docker to be installed.
+    requireDocker.value
+
+    // This task requires that the dependency image be created, and that the main image be staged.
+    mainImageStageDef.value
+    dependencyBuildDef.value
+
+    val logger = Keys.streams.value.log
+    logger.info("Building main image...")
```

Comment:
I've been noticing that when running `sbt generateDockerfile` globally, these generic logging messages have been really hard to localize to a specific project.  It looks like that's the case here too.  Can you add a line signaling what's project is currently building?

Note: Separate change is fine, but this may be an opportunity since you're in here anyway.